test/testdata/deployednettemplates/recipes/fastnet/gen_topology.py

```diff
@@ -0,0 +1,32 @@
+node_types = {"R":6, "N":12, "NPN":6}
```

Comment:
For this round I'm kind of okay with mimicking betanet in terms of composition and distribution. Is this 24 non-co-located nodes? This seems like a lot until we can iron out some details.

@cce Any thoughts on this? I'm kind of okay with the 5 instance (1 relay + 4 partnode) setup to launch at this round while we evaluate our changes.

Reply:
Sure, however you think is best to roll it out, I assume it is relatively easy to update and grow the network as needed

Reply:
OK, reduced to 1 relay, 4 nodes, and 2 NPNs.